### PR TITLE
Updates from video-js-swf 5.0.1

### PR DESCRIPTION
**Pull request by raytiley**
There was an interface change to `get:buffered()`. I honestly have no idea if I implemented it correctly, I just got it to compile. Still haven't tested the binary with latest videojs yet. Will do that soon and will report back. Just wanted to get this up here incase someone could identify any major mistakes I made.

### CODE REVIEW

File: src/com/videojs/providers/HLSProvider.as [hunk 1]
```diff
@@ -254,13 +254,19 @@ package com.videojs.providers{
...
+       /**
+         * Should return an array of normalized time ranges currently
+         * buffered of the media, in seconds.
          */
-        public function get buffered():Number {
-          return _bufferedTime;
+        public function get buffered():Array{
+            if(_bufferedTime) {
+                return [[
+                    _position - _bufferedTime,
```
Comment (mangui) [inline src/com/videojs/providers/HLSProvider.as]: returned array does not seem correct, 
IMHO should be
[[ _position, _bufferedTime]]

if you want to return something even more accurate, taking into account the backbuffer, you can return

[[ _backBufferTime, _bufferedTime]]

you can compute _backBufferTime here https://github.com/mangui/video-js-swf/pull/9/files#diff-f497aef833a75fe7ea0dfa925fa2fbeeR105
with _backBufferTime = event.mediatime.position - event.mediatime.backbuffer;

Comment (mangui) [inline src/com/videojs/providers/HLSProvider.as]: the rest looks good :-)

Comment (raytiley) [inline src/com/videojs/providers/HLSProvider.as]: Your link to where to add `_backBufferTime` doesn't make sense to me... I assume it should go in `HLSProvider` can you tell me which method? Thanks!

**Comment by raytiley** [comment]
I think I figured out the `buffered` implementation.

**Comment by mangui** [comment]
looks good :-)

**Comment by mangui** [comment]
let me know when you get your binary up and running and working

**Comment by codex-corp** [comment]
I'm also trying to get vjs 5 changes , it's working for me 
Here the swf https://github.com/codex-corp/video-js-swf/commit/2e621feb7a62e3abc01d92721356a8ce60410210

**Comment by raytiley** [comment]
My tests yesterday were good. I think this can be merged in.

**Comment by codex-corp** [comment]
@raytiley thanks !

**Pull request merged**